OpenRA.Game/Support/LaunchArguments.cs

```diff
@@ -22,8 +22,11 @@ public class LaunchArguments
 		[Desc("Automatically start playing the given replay file.")]
 		public string Replay;
 
-		[Desc("Dump performance data into cpu.csv and render.csv in the logs folder.")]
-		public bool Benchmark;
+		[Desc("Dump performance data into cpu.csv and render.csv in the logs folder with the given prefix.")]
+		public string Benchmark;
+
+		[Desc("Automatically start playing the given map.")]
+		public string Map;
```

Comment:
This will also be really useful for people working on scripted maps, so good call not merging it with `Benchmark`.

Maybe this will also perk @MustaphaTR or @abcdefg30's interest to review this PR.

Reply:
Fwiw, editing a Lua script does not change the hash (or path), but one can just use the restart map function to reload the script anyway.